packages/cli/src/metadataGeneration/initializer-value.ts

```diff
@@ -48,11 +48,20 @@ export const getInitializerValue = (initializer?: ts.Expression, typeChecker?: t
         nestedObject[p.name.text] = getInitializerValue(p.initializer, typeChecker);
       });
       return nestedObject;
+    case ts.SyntaxKind.ImportSpecifier:
+      const importSpecifier = initializer as ts.ImportSpecifier;
+      const importSymbol = typeChecker.getSymbolAtLocation(importSpecifier.name)!;
```

Comment:
Is there a scenario where we have a program that compiles fine but returns `undefined` here?

Reply:
I pushed a quick fix to just return if it's undefined. Not familiar enough with the Typescript compiler API to know how to deal with this better.

Reply:
I don't see any new commits, am I missing something? If you're handling `undefined`, I'd expect the assertion to be gone:

```suggestion
      const importSymbol = typeChecker.getSymbolAtLocation(importSpecifier.name);
```

Reply:
Sorry looks like I didn't end up pushing the amended commit when I thought I did